elytron/src/main/java/org/wildfly/extension/elytron/SecretKeyCredentialStoreDefinition.java

```diff
@@ -96,7 +97,7 @@ class SecretKeyCredentialStoreDefinition extends AbstractCredentialStoreResource
             .setAttributeGroup(ElytronDescriptionConstants.IMPLEMENTATION)
             .setAllowExpression(true)
             .setDefaultValue(new ModelNode(256))
-            .setAllowedValues(128, 192, 256)
+            .setValidator(new IntAllowedValuesValidator(128, 192, 256))
```

Comment:
I think we could use both allowed values and the validator. One thing that allowed values does is to show the expected values when the user press <TAB>. For this case makes sense to me since it can help the user to think about which values are allowed, they are difficult to identify. 

@lvydra WDYT?

Reply:
Hi @yersan, as I tested it now, it seems that the validator offers the same functionality in terms of showing expected values.

Reply:
Ahh, great then, thanks for checking it